en/landing.ftl

```diff
@@ -9,6 +9,7 @@ hero-section-body = Our secure, easy-to-use email and phone masks help keep your
     private so you can sign up for new accounts anonymously, stop spam texts and junk calls, and 
     get only the emails you want in your inbox.
 hero-section-cta = Get started
+hero-section-social-proof = As seen in
```

Comment:
What is followed after this?

Reply:
@peiying2 sorry, here is the context: 

![image](https://user-images.githubusercontent.com/3924990/232153876-5b125578-106b-4d95-bacb-8be9efaeebab.png)

Reply:
Thank you for the screenshot. Please add a comment by saying something like "website magazines such as xyz, etc". We want the localizers to be able to see it.